api/app/jobs/viirs_snow.py

```diff
@@ -135,6 +159,49 @@ async def _save_clipped_snow_coverage_mosaic_to_s3(self, for_date: date, path: s
                 await client.put_object(Bucket=bucket,
                                     Key=key,
                                     Body=file)
+                
+    def _classify_snow_coverage(self, path: str):
+        source_path = os.path.join(path, RAW_SNOW_COVERAGE_CLIPPED_NAME)
+        source = gdal.Open(source_path, gdal.GA_ReadOnly)
+        source_band = source.GetRasterBand(1)
+        source_data = source_band.ReadAsArray()
+        # Classify the data. Snow coverage in the source data is indicated by values in the range of 0-100. I'm using a range of 
+        # 10 - 100 to increase confidence. In the classified data 1 is assigned to snow covered pixels and all other pixels are 0.
+        classified = np.where((source_data > 10) & (source_data <= 100), 1, 0)
+        output_driver = gdal.GetDriverByName("GTiff")
+        classified_snow_path = os.path.join(path, BINARY_SNOW_COVERAGE_CLASSIFICATION_NAME)
+        classified_snow = output_driver.Create(classified_snow_path, xsize=source_band.XSize, ysize=source_band.YSize, bands=1, eType=gdal.GDT_Byte)
+        classified_snow.SetGeoTransform(source.GetGeoTransform())
+        classified_snow.SetProjection(source.GetProjection())
+        classified_snow_band = classified_snow.GetRasterBand(1)
+        # snow_mask_band.SetNoDataValue(0) ???
```

Comment:
Did you mean to delete this line?

Reply:
Ah, yes, was doing some testing and this line is not required. Thanks!